Core/src/org/sleuthkit/autopsy/datamodel/Accounts.java

```diff
@@ -77,11 +78,16 @@
 public class Accounts extends Observable implements AutopsyVisitableItem {
```

Comment:
since that interface is not public, I can make it public, and then move it into the same unexposed package!  It has more than just accounts, so i will name it _private.